roles/core/dns_server/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# implement more default values which will come from inventory
+j2_current_iceberg_network: ice-0
+external_dns: []
+networks: []
+domain_name: .localhost
+
+start_services: true
+enable_services: true
```

Comment:
I do not approve default usage, please see https://github.com/bluebanquise/bluebanquise/pull/163

Reply:
ok, then I will move them in the converge.yml as pre_tasks, ok?

Reply:
Yes, as long as these variables are not "seen" by the ansible-playbook or the ansible-inventory commands, it is ok :-)